### ISSUE TITLE
Tabbing to "Search PyPI" text box does not give it focus

### ISSUE THREAD

**Issue by zooba** [Microsoft/PTVS#2969]
When you tab in the Python Environments window to the text box to search PyPI, the box highlights but you cannot enter text.

This also affects the "Install Python Package" command.
